packages/svelte/test/validator/samples/indeterminate-attribute/errors.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
This test in unrelated to the `svelte/elements` type definition.

Reply:
Thanks for the feedback I just moved this and changed the test format.  Please let me know if other changes are desired!

Reply:
There is no test for element type definition. So removing it should be fine. This is not the right one as well.

Reply:
Cool. Removed.